api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java

```diff
@@ -64,4 +64,16 @@
     @JsonProperty("phoneNumber")
     @ApiModelProperty(notes = "The claimant's UK phone number. Must be in +44 format, e.g. +447123456789")
     private String phoneNumber;
+
+    // See https://stackoverflow.com/questions/201323/how-to-validate-an-email-address-using-a-regular-expression for explanation of email regex.
+    @NotNull
+    @Pattern(regexp = "(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|\"(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21\\x23-\\x5b\\x5d-\\x7f]"
```

Comment:
suggest we have a single constant for this regex

Reply:
Done.